helper/helper.py

```diff
@@ -0,0 +1,124 @@
+# This script helps synchronize contents from their respective sources of
+# truth (usually GitHub repositories of each Tekton
+# components, such as tektoncd/pipelines) to tektoncd/website.
```

Comment:
sounds like the "helper" is intended to sync the docs contents - could we name it something a bit more descriptive, e.g. `sync` or `docs-update` or `fetcher`?

Reply:
Got it.